pkg/httphelper/client.go

```diff
@@ -324,6 +324,78 @@ func (client *NodeMgmtClient) CallCreateRoleEndpoint(pod *corev1.Pod, username s
 	return nil
 }
 
+// Create a new superuser with the given username and password
+func (client *NodeMgmtClient) CallDropRoleEndpoint(pod *corev1.Pod, username string) error {
```

Comment:
**Todo:** Could you add a test for this deletion endpoint? We have one for the list endpoint only it seems.

Reply:
Done (for all 3 methods, even the one I didn't add in this PR).